docs/samples/connecting-gateway-odh-rhoai/setup-gateway-prerequisites.sh

```
@@ -0,0 +1,610 @@
+      -o jsonpath='{.spec.template.spec.containers[0].args}' 2>/dev/null || echo "")
+
+    if echo "$current_args" | grep -q "cluster-audience"; then
+      echo "  Fixing maas-controller args (removing unsupported --cluster-audience flag)..."
+      kubectl annotate deployment maas-controller -n "$NAMESPACE" \
+        opendatahub.io/managed=false --overwrite
+      kubectl patch deployment maas-controller -n "$NAMESPACE" --type='json' -p='[
+        {"op":"replace","path":"/spec/template/spec/containers/0/args",
+         "value":["--leader-elect","--health-probe-bind-address=:8081",
+                  "--gateway-name=$(GATEWAY_NAME)","--gateway-namespace=$(GATEWAY_NAMESPACE)",
+                  "--maas-api-namespace=$(MAAS_API_NAMESPACE)",
+                  "--maas-subscription-namespace=$(MAAS_SUBSCRIPTION_NAMESPACE)"]}]'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)